meshtastic/__main__.py

```diff
@@ -350,6 +350,26 @@ def onConnected(interface):
                 print(f"Setting device owner short to {args.set_owner_short}")
             interface.getNode(args.dest, False, **getNode_kwargs).setOwner(long_name=args.set_owner, short_name=args.set_owner_short)
 
+        if args.set_is_unmessageable:
```

Comment:
Similarly to the comment in node.py, this block should be merged into the `if args.set_owner or args.set_owner_short` above it, so everything's done in one admin command to the node.